production/helm/loki/values.yaml

```diff
@@ -267,6 +272,9 @@ loki:
   # --  Optional querier configuration
   querier: {}
 
+  # --  Optional ingester configuration
+  ingester: {}
```

Comment:
I was about to debate adding this but saw that we have `querier`. So it's all good.